docs/preview/02-Features/powershell/azure-devops.md

```diff
@@ -88,7 +88,7 @@ This function is intended to be used from an Azure DevOps pipeline. Internally,
 One of the environment variables that is used, is the `SYSTEM_ACCESSTOKEN` variable. However, due to safety reasons this variable is not available out-of-the box.
 To be able to use this variable, it must be explicitly added to the environment-variables.
 
-> ⚠ When you are using a Linux agent, you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.
+> ⚠ When you are using a Linux agent (which is recommended), you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.
```

Comment:
Using a Linux agent is not always possible.   For instance when you want to execute a DacPac task, or if you want to build a Windows container.
I'd change this to something like this:

```suggestion
> ⚠ When you are using a Linux agent, you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.

> We have seen a much better performance when using Linux agents, and would recommend using Linux agents when possible.
```